maskrcnn_benchmark/utils/metric_logger.py

```diff
@@ -61,3 +65,42 @@ def __str__(self):
                 "{}: {:.4f} ({:.4f})".format(name, meter.median, meter.global_avg)
             )
         return self.delimiter.join(loss_str)
+
+
+class TensorboardLogger(MetricLogger):
+    def __init__(self,
+                 log_dir='logs',
+                 exp_name='maskrcnn-benchmark',
```

Comment:
Can we pass a single argument here, instead of `log_dir` and `exp_name`, pass and `save_file` or something?
There is no real value in having both, given that they are immediately concatenated into a single string.